example/map-halogen/src/Main.purs

```diff
@@ -52,7 +54,7 @@ ui =
       [ HH.slot MapSlot mapComponent unit $ Just <<< H.action <<< HandleMapUpdate
       , HH.button
           [ HP.class_ (HH.ClassName "goto")
-          , HE.onClick (HE.input_ $ GoTo $ makeLngLat 44.81647122397245 41.661632116606455) 
+          , HE.onClick (HE.input_ $ GoTo $ LngLat.make { lng: 44.81647122397245, lat: 41.661632116606455 })
           ]
           [ HH.text "GoTo Tbilisi" ]
```

Comment:
It's funny to read this ^ in every place :)

Reply:
🕺